models/requestduplicate.js

```diff
@@ -0,0 +1,30 @@
+/* Smoke Alarm Installation Request Portal (getasmokealarm.org)
+ * 
+ * Copyright (C) 2015  American Red Cross
```

Comment:
Also 2017.  (I assume these are just copy/pasto's, no problem.)

Reply:
Yep, I'd just copied and pasted these, thanks for catching it!